shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -2081,4 +2102,15 @@ public boolean isFailure() {
         }
     }
 
+    @Data
+    public static final class Flush implements WireCommand {
+        final WireCommandType type = WireCommandType.FLUSH_SEGMENT;
+        final UUID writerId;
+        final String segment;
```

Comment:
Same

Reply:
yes, see the setupSegments table;  Entry is a pair of <writer id, segment>.